### ISSUE TITLE
Error when use provider_name = TENANT on 0.6.0 mongodbatlas provider version.

### ISSUE THREAD

**Issue by robsonmantovani** [mongodb/terraform-provider-mongodbatlas#246]
This error only occurs when i user 0.6.0 version from mongodbatlas provider, on 0.5.1 works fine.

```sh
Error: error reading MongoDB Cluster (mongoexamplecluster): GET https://cloud.mongodb.com/api/atlas/v1.0/groups/5ee7f662eae0c80ddc83b3f0/containers?providerName=TENANT: 404 (request "Not Found") Provider TENANT not currently supported.

  on ../main.tf line 9, in resource "mongodbatlas_cluster" "cluster":
   9: resource "mongodbatlas_cluster" "cluster" {
```
To reproduce you can use TENANT example. 

```
resource "mongodbatlas_cluster" "cluster-test" {
  project_id              = "<YOUR-PROJECT-ID>"
  name                    = "cluster-test-global"
  //M2 must be 2, M5 must be 5
  disk_size_gb            = "2"

  //Provider Settings "block"
  provider_name = "TENANT"
  backing_provider_name = "AWS"
  provider_region_name = "US_EAST_1"
  provider_instance_size_name = "M2"

  //These must be the following values
  mongo_db_major_version = "4.2"
  auto_scaling_disk_gb_enabled = "false"
}
```

**Comment by themantissa** [comment]
@robsonmantovani thank you for reporting - we'll investigate and report back shortly.

**Comment by DenisBiondic** [comment]
Same issue with Shared clusters on Azure

**Comment by themantissa** [comment]
Issue has been found - fix is in progress. We expect to release a fix for this and for another bug with 0.6.1.  fyi @PacoDw

**Comment by PacoDw** [comment]
Hello, guys, I added the #247 PR that resolves the issues, so could you check if your issue persists in that PR? the branch is `fix-#246`

Thank you so much! Let me know if you have another comment or concern.

**Comment by pdemagny** [comment]
Hi, same issue with v0.6.0 on GCP.

**Comment by themantissa** [comment]
@pdemagny thank you, we are aware it impacts all Providers.  If any reporters (@robsonmantovani and @DenisBiondic) can test using the branch on #247 to confirm it's addressed that will help a great deal. We'd like to get this fix out ASAP.

**Comment by pdemagny** [comment]
Hi @themantissa. So i tested it and it worked like a charm for me on GCP.
Thank you all for you quick handling of this issue !

**Comment by themantissa** [comment]
Hi all, this is closed as 0.6.1 w/ the fix is built and should be released by HashiCorp officially soon. Thank you for the quick reporting!
